spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> I see. In that case, it violates principle <code>#3</code> from our <a href="https://github.com/estree/estree#philosophy">Philosophy</a> by duplicating data that is already in <code>raw</code>.

`raw` is not part of this specification.

> What can be better instead would be adding <code>directive</code> boolean flag in this extension of <code>ExpressionStatement</code>.

The value must be a string instead of a boolean because otherwise `"use strict"` and `"use\x20strict"` would yield identical ASTs.